### PR TITLE
[9.0][purchase_request_to_procurement] Allow to select all products, even when they are not purchaseable.

### LINKED ISSUE #427: [9.0][purchase_request_to_procurement] Does not allow to add non-purchase products to purchase request
In case that you create a purchase request that should initiate a procurement to transfer some stock, you are sometimes stuck because only products that can be purchased can be selected in the purchase request.

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Fixes #427

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
cc @lreficent @benwillig

**Comment by LoisRForgeFlow** [comment]
I would take the ocassion to also attend [this comment](https://github.com/OCA/purchase-workflow/pull/424#issuecomment-312199138).

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by MiquelRForgeFlow — APPROVED** [review]
nice :+1:

File: purchase_request/models/purchase_request.py [hunk 1]
```diff
@@ -255,7 +255,7 @@ def _compute_supplier_id(self):
                                      'Procurement Order',
                                      readonly=True, copy=False)
     cancelled = fields.Boolean(
-        string="Cancelled", readonly=True, default=False)
+        string="Cancelled", readonly=True, default=False, copy=False)
```
Comment (benwillig) [inline purchase_request/models/purchase_request.py]: Why removing ```copy=False``` from the field declaration?

**Review by benwillig — APPROVED** [review]

**Pull request merged**